Hints.py

```diff
@@ -349,6 +349,9 @@ def color_text(gossip_text: GossipText) -> str:
     colors = list(gossip_text.colors) if gossip_text.colors is not None else []
     color = 'White'
 
+    while text.count('#') > 2 * len(colors):
+        colors[:0] = [colors[0]]
```

Comment:
I feel like this should be handled in the function generating the hint text for Sometimes hints instead of here, as other text boxes might need a different behavior.

Reply:
I'll check that out once I have a little more time.
I think it's fine to handle it here to have a default in case we got out of bounds for colors like I accidentally did ; but to also handle it directly in the sometimes themselves for a more controlled concept.

Hard to decide.

Reply:
We already have a default: It keeps using the last color for extra placeholders. I don't think there's a good reason to change it to behaving this way other than for Sometimes hints.

Reply:
If we have a default then i have no objections, i'll change that later.